remote-api/src/main/resources/alfresco/web-scripts-application-context.xml

```diff
@@ -249,6 +249,7 @@
       <property name="delegate" ref="webscript.content.streamer" />
       <property name="contentService" ref="contentService" />
       <property name="repository" ref="repositoryHelper" />
+      <property name="nonAttachContentTypes" value="#{T(java.util.Arrays).asList('${content.nonAttach.mimetypes}')}" />
```

Comment:
Please double-check the differences with the implementation shown [here](https://github.com/Alfresco/alfresco-enterprise-repo/pull/819/files#diff-be731d09427ccf7e6a2346777d6fdc801b5ef77d5c9459476ded44d396fa9fdbR63). The implementation that I linked defaults to an empty list. We don't want `content.nonAttach.mimetypes` to be unnecessarily mandatory.